DQMOffline/Trigger/plugins/PhotonMonitor.cc

```diff
@@ -0,0 +1,350 @@
+#include "DQMOffline/Trigger/plugins/PhotonMonitor.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DQM/TrackingMonitor/interface/GetLumi.h"
+
+#include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
+
+double MAX_PHI1 = 3.2;
```

Comment:
hi @gomber 
no global variables - just move these into the scope of the Photon Monitor class